homeassistant/components/netatmo/media_source.py

```diff
@@ -95,7 +99,7 @@ def _build_item_response(
         )
 
         if not media.can_play and not media.can_expand:
-            return None
+            raise IncompatibleMediaSource
```

Comment:
This case can happen according to the test data if there's a camera event with an event id but no media url so the event can't be played.